components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Settings.php

```diff
@@ -44,6 +44,12 @@ public function __construct() {
 		// add the same tab for taxonomies
 		add_filter( 'pods_admin_setup_edit_tabs_taxonomy', array( $this, 'tab' ), 11, 3 );
 
+		// add the same tab for comments
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 420d29538b0164e21fef6e8380ea7b48</sub>

Reply:
```suggestion
		// Add the same tab for comments.
```